component-server-parent/component-server/src/main/java/org/talend/sdk/component/server/front/BulkReadResourceImpl.java

```diff
@@ -195,6 +198,7 @@ private CompletableFuture<BulkResponses.Result> doExecute(final BulkRequests.Req
         final CompletableFuture<BulkResponses.Result> promise = new CompletableFuture<>();
         final InMemoryResponse response = new InMemoryResponse(() -> true, () -> {
             result.setResponse(outputStream.toByteArray());
+            result.setResponseString(new String(outputStream.toByteArray()));
```

Comment:
Here we explicitly response with a string but expected type should be json as described in the API (https://github.com/Talend/component-runtime/blob/ee8fd4afd5a5dbb091b5b4dff900653f31a78d63/component-server-parent/component-server-api/src/main/java/org/talend/sdk/component/server/api/BulkReadResource.java#L37). We may perhaps have bad escaped value...